webpack.config.common.js

```diff
@@ -13,8 +13,11 @@ const config = {
   resolve: {
     extensions: [".js", ".jsx"],
     alias: {
-      styles: path.resolve(__dirname, "app/assets/src/styles"),
-      "~": path.resolve(__dirname, "app/assets/src")
+      "~": path.resolve(__dirname, "app/assets/src"),
+      api: path.resolve(__dirname, "app/assets/src/api"),
```

Comment:
How do you feel about using `~ui` as the alias instead of `ui`? Makes it more clear that it's an alias.

`styles` doesn't have the `~` because it's primarily used in `scss`, which automatically appends the `~`.

Reply:
I can do that. Might also avoid some conflicts later on.

Reply:
Sounds good. Although...how are there other files if you just added the aliases?

Reply:
Fair point, i added this a long time ago... but i guess it was never included in a PR. So, I can update it.